<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_PG_AssessmentReport_Add</class>
 <widget class="QDialog" name="Frm_PG_AssessmentReport_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加/编辑毁伤评估报告</string>
  </property>
  <widget class="QGroupBox" name="gb_report_info">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>541</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>报告基本信息</string>
   </property>
   <widget class="QLabel" name="lbl_report_code">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>报告编号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_report_code">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_report_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>报告名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_report_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>441</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_required_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>13</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 0, 0, 255)</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_required_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>13</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 0, 0, 255)</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_association_info">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>129</y>
     <width>541</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>关联信息</string>
   </property>
   <widget class="QLabel" name="lbl_assessment_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>73</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>毁伤评估ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_scene_name">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>场景名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_scene_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_parameter_id">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>参数ID：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_parameter_id">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ammunition_name">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>弹药名称：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_target_type">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>目标类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_target_type">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <item>
     <property name="text">
      <string>机场跑道</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>单机掩蔽库</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>地下指挥所</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="ed_ammunition_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_target_name">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>目标名称：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_target_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_assessment">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_required_1">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>20</y>
      <width>13</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>13</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 0, 0, 255)</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_assessment_id">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_assessment_result">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>281</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>评估结果</string>
   </property>
   <widget class="QLabel" name="lbl_damage_degree">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>毁伤等级：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_damage_degree">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <item>
     <property name="text">
      <string>未达到轻度毁伤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>轻度毁伤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中度毁伤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重度毁伤</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>完全摧毁</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_comment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>评估结论：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_comment">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_personnel_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>报告人员信息</string>
   </property>
   <widget class="QLabel" name="lbl_creator_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>创建用户：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_creator_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>25</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color: #f0f0f0;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_reviewer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>25</y>
      <width>48</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>审核人：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ed_reviewer">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>25</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
